specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2023-05-01/cognitiveservices.json

```diff
@@ -3692,6 +4146,83 @@
       },
       "description": "Cognitive Services account commitment period."
     },
+    "ModelListResult": {
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of Model.",
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Model"
+          },
+          "x-ms-identifiers": [
+            "kind",
+            "skuName",
+            "/model/name",
+            "/model/format",
+            "/model/version"
+          ],
+          "description": "Gets the list of Cognitive Services accounts Model and their properties."
+        }
+      },
+      "description": "The list of cognitive services models."
+    },
+    "Model": {
+      "description": "Cognitive Services Model.",
+      "type": "object",
+      "properties": {
+        "model": {
+          "type": "object",
+          "description": "Model Metadata.",
+          "$ref": "#/definitions/AccountModel"
+        },
+        "kind": {
+          "$ref": "#/definitions/Kind",
+          "description": "The Kind of the Model."
+        },
+        "skuName": {
+          "$ref": "#/definitions/SkuName",
+          "description": "The SKU of the Model."
+        }
+      }
+    },
+    "ModelSku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the model SKU."
+        },
+        "usageName": {
+          "type": "string",
+          "description": "The usage name of the model SKU."
+        },
+        "unitsPerInstance": {
```

Comment:
This contact has changed. https://msazure.visualstudio.com/Cognitive%20Services/_git/Platform-ResourceProvider?path=%2Fsrc%2FCommon%2FResourceProvider.Contracts%2FDeployments%2FModelSku.cs&_a=contents&version=GBmaster